### DISCUSSION TITLE
auto convertion from tui to web ui

### DISCUSSION THREAD

**Discussion by nevzatseferoglu** [charmbracelet/bubbletea#750] [Ideas]
I was just thinking how awesome it would be to have an automatic generation feature that could convert Bubbletea UI into web UI. Are there any ongoing efforts or community-driven repositories focused on accomplishing this type of task ?

**Comment by bashbunni** [comment]
Hey @nevzatseferoglu this is definitely something I've seen asked more than once. I'm going to convert this to a discussion. There's an existing discussion about WASM support which aims to accomplish a similar goal of making Bubble Tea available in the browser https://github.com/charmbracelet/bubbletea/discussions/300
